src/subdomains/core/history/controllers/transaction.controller.ts

```diff
@@ -310,11 +310,15 @@ export class TransactionController {
     if (transaction.targetEntity?.cryptoInput?.txType === PayInType.PAYMENT)
       throw new BadRequestException('You cannot refund payment transactions');
 
-    const feeAmount = transaction.targetEntity.cryptoInput
+    const dfxFeeAmount = transaction.targetEntity.cryptoInput
       ? await this.feeService.getBlockchainFee(transaction.targetEntity.cryptoInput.asset, false)
       : 0;
```

Comment:
Shouldn't this be network fee instead of dfx?

Reply:
Hehe yeah your are right 😁